sites/svelte-5-preview/src/routes/docs/content/03-appendix/02-breaking-changes.md

```diff
@@ -203,3 +222,11 @@ Previously, bindings did not take into account `reset` event of forms, and there
 ### `walk` not longer exported
 
 `svelte/compiler` reexported `walk` from `estree-walker` for convenience. This is no longer true in Svelte 5, import it directly from that package instead in case you need it.
+
+### Content inside `svelte:options` is forbidden
+
+In Svelte 4 you could have content inside a `<svelte:options />` tag. It was ignored, but you could write something in there. In Svelte 5, content inside that tag is a compiler error.
+
+### Slots inside templates are preserved
```

Comment:
Maybe `Slots inside <template> are preserved` for more clarity?

Reply:
```suggestion
### `<slot>` elements in declarative shadow roots are preserved
```